app/views/build_estimates/_sidebar_with_date.html.slim

```diff
@@ -1,8 +1,10 @@
 = render 'layouts/sidebar'
-  h2.govuk-heading-s  = t("generic.date")
-  p.govuk-body-m = Date.current.strftime('%d %B %Y')
+  h2.govuk-body-m
+    strong = t("generic.date")
+    p.govuk-body-m = Date.current.strftime('%d %B %Y')
 
   - if (block = yield).present?
-    h2.govuk-heading-s = t("generic.related_content_title")
-    ul.govuk-list
-      = block
\ No newline at end of file
+    h2.govuk-body-m
+      strong = t("generic.related_content_title")
+      ul.govuk-list
+        = block
```

Comment:
Really nitpicky, but each link in the designs has a margin bottom of 20px. Ours don't (it's just not very noticeable yet because we don't have any example pages with multiple links in the sidebar).